code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -272,6 +272,7 @@
 	modules += new /obj/item/weapon/revivalprod(src)
 	modules += new /obj/item/weapon/inflatable_dispenser/robot(src)
 	modules += new /obj/item/robot_rack/bed(src)
+	modules += new /obj/item/weapon/cookiesynth/lollipop(src)
```

Comment:
might want to add it to syndicate mediborgs too

Reply:
poop

Reply:
okay done